src/main/config/config_eeprom.c

```diff
@@ -92,6 +243,28 @@ void initEEPROM(void)
 
     STATIC_ASSERT(sizeof(configFooter_t) == 2, footer_size_failed);
     STATIC_ASSERT(sizeof(configRecord_t) == 6, record_size_failed);
+
+#ifdef EEPROM_IN_FILE
+    void loadEEPROMFromFile(void);
+
+    loadEEPROMFromFile();
+#endif
+
+#ifdef EEPROM_IN_EXTERNAL_FLASH
```

Comment:
Should be `#elsif` if they can't be combined.

Reply:
currently that is the case, I will fix.